doc/content.tex

```diff
@@ -1,1347 +0,0 @@
-    value. Then it checks if there are significant values of ACF present at following $n*period$ indices.
-    There have to be present at lease two consecutive values of ACF, so $n$ takes values from $1,2,3\dots$
-
-    \begin{algorithm}
-        \caption{Find period of time series} \label{alg:period-find}
-        \begin{algorithmic}[1]
-        \Function{findFrequency}{$int[] ts$}
-            \If{$unitRootPresent(ts)$} \Comment{e.g. ADF test}
-                \State $ts \gets diff(ts)$ \Comment{first order differences}
-            \EndIf
-            \State $acf \gets acf(ts)$ \\
-                        \Comment{returns index of the highest value}
-            \State $period \gets findHighest(ts, period)$
-            \While{$period * 2 < ts.length$}
-                \If{$checkPeriodExists(x, ts)$}
-                    \State \Return $period$
-                \EndIf
-              \State $period \gets findHighest(ts, period)$
-            \EndWhile
-            \State \Return $1$
-        \EndFunction
-        \end{algorithmic}
-    \end{algorithm}
-
-%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
-\chapter{Analytical Forecasting Process} \label{chap:models-demonstration}
-In the previous chapter several time series models were described. However, in Hawkular only a few of them were
-selected and implemented.
-
-This chapter demonstrates targeted models on real time series. There is also conducted a
-statistical comparison of the models. So this chapter also contains theory how different models should be compared. At
-the end there is a discussion which models were selected for usage in Hawkular.
-
-    %%%%%%%%
-    \section{Evaluating Forecasting Accuracy} \label{sec:mse-mae}
-    In order to evaluate a forecasts produced by particular model it is important to calculate the errors of the
-    forecasts. There are several statistics for evaluating forecasting accuracy. The most used ones are mean squared
-    error (MSE) \ref{eq:accuracy-mse} and mean absolute error (MAE) \ref{eq:accuracy-mae} \cite{statistika}. The
-    difference between them is that MSE emphasizes the extremes while MAE is more robust to outliers
-    \cite{hyndman-forecasting}.
-
-    \begin{gather} \label{eq:accuracy-mse}
-         MSE = \frac{1}{n} \sum_{i=1}^{n}(y_i - \hat{y_i}) \\
-         MAE = \frac{1}{n} \sum_{i=1}^{n} \abs{y_i - \hat{y_i}} \label{eq:accuracy-mae}
-    \end{gather}
-
-    %%%%%%%%
-    \section{Measuring Model Quality} \label{sec:model-quality}
-    When comparing multiple models, statistics like MSE or MAE are not the best objective functions. Comparison
-    based on this statistics can select complicated model with lots of parameters, which may overfits training data
-    and most importantly it selects less robust model \cite{cipra}. Therefore for comparison multiple models another
-    factors has to be added to the objective function. These factors are number of parameters of the model.
```

Comment:
...factors have to be added.../namiesto has (jedine, ze ten faktor je len jeden, vtedy factor namiesto factors :-))